libs/estdlib/src/erlang.erl

```diff
@@ -223,13 +223,8 @@ send_after(Time, Dest, Msg) ->
 %%
 %% @end
 %%-----------------------------------------------------------------------------
--spec process_info
-    (Pid :: pid(), heap_size) -> {heap_size, non_neg_integer()};
-    (Pid :: pid(), total_heap_size) -> {total_heap_size, non_neg_integer()};
-    (Pid :: pid(), stack_size) -> {stack_size, non_neg_integer()};
-    (Pid :: pid(), message_queue_len) -> {message_queue_len, non_neg_integer()};
-    (Pid :: pid(), memory) -> {memory, non_neg_integer()};
-    (Pid :: pid(), links) -> {links, [pid()]}.
+-spec process_info(Pid :: pid(), Key :: atom()) ->
+    {LookupKey :: atom(), Value :: non_neg_integer()} | {links, [LinkPid :: pid()]}.
```

Comment:
are we sure about this change?

Reply:
I don't mind reverting this change, but I think the more concise version looks better, while still conveying all of the correct information.

Reply:
This change doesn't look just cosmetic to me, we are changing to a less specific typespec.
CC @fadushin or @pguyot

Reply:
Just as a point of reference here is the OTP spec:

`process_info(Pid, Item) -> InfoTuple | [] | undefined`
`process_info(Pid, ItemList) -> InfoTupleList | [] | undefined`

Reply:
It might be better if I change this to closer to the OTP definition, and add a trypespec for InfoTuple.